Advanced Examples/React/ci/__main__.py

```diff
@@ -0,0 +1,149 @@
+# Enthought product code
+#
+# (C) Copyright 2010-2022 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This file and its contents are confidential information and NOT open source.
+# Distribution is prohibited.
+
+"""
+    This is the "ci" module for the React example.
+"""
+
+import click
+import os.path as op
+import subprocess
+import sys
+import os
+import json
+
+SRC_ROOT = op.abspath(op.join(op.dirname(__file__), ".."))
+
+# Docker image will be tagged "IMAGE:VERSION"
+IMAGE = "quay.io/enthought/edge-native-app-flask-demo"
+VERSION = "1.1.0"
+
+# These will go into the built Docker image.  You may wish to modify this
+# minimal example to pin the dependencies, or use a bundle file to define them.
+APP_DEPENDENCIES = [
+    "enthought_edge>=2.6.0",
+    "appdirs",
+    "packaging",
+    "pip",
+    "pyparsing",
+    "setuptools",
+    "six",
+    "click",
+    "flask>2",
+    "gunicorn",
+    "opencv_python",
+]
+
+# This will be used when running locally ("run" or "preflight" commands).
+# We just use the last component of the full image URL.
+CONTAINER_NAME = IMAGE.split("/")[-1]
+
+
+@click.group()
+def cli():
+    """Group for Click commands"""
+    pass
+
+
+@cli.command()
+@click.option("--rebuild-zbundle", default=False, is_flag=True)
+def build(rebuild_zbundle):
+    """Build the Docker image"""
+
+    # First, build the React application
+    jsdir = op.join(SRC_ROOT, "src", "application", "frontend")
+    subprocess.run(["npm", "install"], cwd=jsdir, check=True)
```

Comment:
The recommended practice is to use `npm ci` instead of `npm install` when doing a production build